web/tsconfig.json

```diff
@@ -64,7 +64,7 @@
 
     /* Advanced Options */
     "skipLibCheck": true,                     /* Skip type checking of declaration files. */
-    "forceConsistentCasingInFileNames": true, /* Disallow inconsistently-cased references to the same file. */
+    "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
```

Comment:
JavaScript has a really weird convention to use trailing commas.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

Reply:
This is JSON "JSON, however, disallows trailing commas."
, so trailing commas should not be in here. Also throws an error in my IDE

Reply:
Ohh shoot good catch

Reply:
Trailing commas should just stop being a thing 😆